docs/operation-guides/config-07-kiali-grafana.md

```diff
@@ -44,3 +44,247 @@ Kyma does not expose Kiali and Grafana by default. However you can still access
+
+  ```yaml
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: oauth2-grafana
+    labels:
+      app: oauth2-grafana
+      target: oauth2-grafana
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: oauth2-grafana
+    template:
+      metadata:
+        labels:
+          app: oauth2-grafana
+      spec:
+        containers:
+        - name: oauth2-proxy
+          image: quay.io/oauth2-proxy/oauth2-proxy:v7.1.3
+          imagePullPolicy: IfNotPresent
+          args:
+          - --provider=github
+          - --email-domain="*"
+          - --http-address=0.0.0.0:3000
+          - --upstream=http://monitoring-grafana.kyma-system.svc:80
+          - --cookie-name=grafana_oauth2_proxy
+          - --proxy-prefix=/oauth2
+          - --ping-path=/oauth2/healthy
+          - --silence-ping-logging=true
+          - --reverse-proxy=true
+          - --skip-provider-button=true
+          - --cookie-secure
+          envFrom:
+          - secretRef:
+              name: oauth2-grafana-secret
+          ports:
+          - name: http
+            containerPort: 3000
+            protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /oauth2/healthy
+              port: http
+            initialDelaySeconds: 3
+            timeoutSeconds: 2
+          readinessProbe:
+            httpGet:
+              path: /oauth2/healthy
+              port: http
+            initialDelaySeconds: 3
+            timeoutSeconds: 2
+        securityContext:
+          fsGroup: 65534
+          runAsNonRoot: true
+          runAsUser: 65534
+  ```
+
+  5. Create a Service for the `oauth2_proxy`:
+
+  ```yaml
+  apiVersion: v1
+  kind: Service
+  metadata:
+    name: oauth2-grafana
+    labels:
+      app: oauth2-grafana
+  spec:
+    type: ClusterIP
+    ports:
+    - port: 3000
+      name: http
+      protocol: TCP
+      targetPort: http
+    selector:
+      app: oauth2-grafana
+  ```
+
+  6. Expose the Service via Istio by creating a VirtualService. Set the domain in the `hosts` list to your desired name:
+
+  ```yaml
+  apiVersion: networking.istio.io/v1alpha3
+  kind: VirtualService
+  metadata:
+    name: oauth2-grafana
+  spec:
+    hosts:
+    - grafana.kyma.example.com
+    gateways:
+    - kyma-system/kyma-gateway
+    http:
+    - match:
+      - uri:
+          regex: /.*
+      route:
+      - destination:
+          port:
+            number: 3000
+          host: oauth2-grafana
+  ```
+  </details>
+</div>
+
+The `oauth2_proxy` supports a wide range of other well known authentication services or OpenID Connect for custom solutions. See the [list of supported providers](https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/oauth_provider) to find instructions for other authentication services.
```

Comment:
```suggestion
>**NOTE:** The `oauth2_proxy` supports a wide range of other well-known authentication services or OpenID Connect for custom solutions. See the [list of supported providers](https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/oauth_provider) to find instructions for other authentication services.
```

I wonder whether this information would be better above the step description, as it explains why we call our instructions an "example".  Wouldn't readers want to know this first; before reading all the steps?

Reply:
Good suggestion. Will move it.